Library/Formula/cvc4.rb

```diff
@@ -0,0 +1,41 @@
+class Cvc4 < Formula
+  homepage "http://cvc4.cs.nyu.edu/"
+  url "http://cvc4.cs.nyu.edu/builds/src/cvc4-1.4.tar.gz"
+  sha256 "76fe4ff9eb9ad7d65589efb47d41aae95f3191bd0d0c3940698a7cb2df3f7024"
+
+  devel do
+    url "http://cvc4.cs.nyu.edu/builds/src/unstable/latest-unstable.tar.gz"
+    version "1.5-prerelease"
+  end
+
+  depends_on "boost" => :build
+  depends_on "gmp"
+  depends_on "libantlr3c" => ["with-exceptions"]
```

Comment:
I'd be fine making exceptions the default for libantlr3c; it looks like no other formulae in the main Homebrew repo depend on it, so it hopefully wouldn't break existing workflows.